config/docker-testnet.config.src

```diff
@@ -23,6 +23,7 @@
      {network, testnet},
      {api_base_url, "https://testnet-api.helium.wtf/v1"},
      {jsonrpc_ip, {0,0,0,0}}, %% bind jsonrpc to host when in docker container
+     {gateway_and_mux_enable, ${GW_AND_MUX}},
```

Comment:
I think adding these ENV vars (ones that cause the sys.config to become invalid if they're not supplied) is probably a bad idea, or at least one that should be re-examined. At a minimum putting quotes around the vars would mean, at worst, that the var would be set to the empty string?